ee/candi/cloud-providers/openstack/layouts/standard/base-infrastructure/main.tf

```diff
@@ -5,6 +5,7 @@ module "network_security" {
   source           = "../../../terraform-modules/network-security"
   prefix           = local.prefix
   remote_ip_prefix = "0.0.0.0/0"
```

Comment:
Can we get rid of `remote_ip_prefix` variable? I think it is confusing if we have `ssh_allow_list`.

Reply:
Yes, that's a good point.